airbyte-integrations/connectors/source-tmdb/source_tmdb/tmdb.yaml

```diff
@@ -0,0 +1,295 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://api.themoviedb.org/3/"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "apikey"
+      api_token: "{{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    schema_loader:
+      type: JsonSchema
+      file_path: "./source_tmdb/schemas/{{ options['name'] }}.json"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+  certification_movie_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "certification_movie"
+      path: "/certification/movie/list?api_key={{ config['api_key'] }}"
+
+  certification_tv_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "certification_tv"
+      path: "/certification/tv/list?api_key={{ config['api_key'] }}"
+
+  changes_movie_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "changes_movie"
+      path: "/movie/changes?api_key={{ config['api_key'] }}&page=1"
+
+  changes_tv_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "changes_tv"
+      path: "/tv/changes?api_key={{ config['api_key'] }}&page=1"
+
+  changes_person_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "changes_person"
+      path: "/person/changes?api_key={{ config['api_key'] }}&page=1"
+
+  movies_details_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "movies_details"
+      path: "/movie/{{ config['movie_id'] or '550' }}?api_key={{ config['api_key'] }}"
```

Comment:
I don't think this a good way, make the `movie_id` required and remove the default value.

Reply:
Done